defaults/main.yml

```diff
@@ -54,11 +54,18 @@ postgresql_pg_hba_default:
   - { type: local, database: all, user: all, address: "",             method: "{{ postgresql_default_auth_method }}", comment: '"local" is for Unix domain socket connections only' }
   - { type: host,  database: all, user: all, address: "127.0.0.1/32", method: "{{ postgresql_default_auth_method }}", comment: "IPv4 local connections:" }
   - { type: host,  database: all, user: all, address: "::1/128",      method: "{{ postgresql_default_auth_method }}", comment: "IPv6 local connections:" }
+  - { comment: "Local root Unix user, passwordless access", type: local, database: all, user: {{ postgresql_admin_user }}, address: "", method: "peer map=root_as_{{ postgresql_admin_user }}" }
```

Comment:
Can you write comment at the end like in lines above? It will look more consistent for a reader.